game.py

```diff
@@ -20,8 +20,34 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses  = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
 
+    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES  :
```

Comment:
```suggestion
    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES:
```

Remove 2 extra whitespaces to keep your code neat and professional.